src/style.css

```diff
@@ -1,3 +1,46 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 100px;
+  width: 1024px;
+  top: 37px;
+  left: -601px;
+  border: 1px solid rgb(248, 245, 245);
+}
+
+.logo {
+  margin: 0 20px;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 20px;
+  list-style: none;
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+a {
+  text-decoration: none;
+  background: #ffff;
```

Comment:
The background color '#ffff' is missing a character. It should be '#ffffff' for white.